.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules
 coverage
 bench/.results
 types/generated.d.ts
+
+# VS Code configuration
+.vscode
```

Comment:
You should add this to your machine's `.gitignore`, rather than adding it in this repo.